orc/orcprogram-neon.c

```diff
@@ -854,10 +1082,16 @@ orc_neon_emit_loop (OrcCompiler *compiler, int unroll_index)
     if (compiler->vars[k].vartype == ORC_VAR_TYPE_SRC ||
         compiler->vars[k].vartype == ORC_VAR_TYPE_DEST) {
       if (compiler->vars[k].ptr_register) {
-        orc_arm_emit_add_imm (compiler,
-            compiler->vars[k].ptr_register,
-            compiler->vars[k].ptr_register,
-            compiler->vars[k].size << compiler->loop_shift);
+        if (compiler->is_64bit)
+          orc_arm64_emit_add_imm (compiler, 32,
+              compiler->vars[k].ptr_register,
+              compiler->vars[k].ptr_register,
+              compiler->vars[k].size << compiler->loop_shift);
+        else
+          orc_arm_emit_add_imm (compiler,
+              compiler->vars[k].ptr_register,
+              compiler->vars[k].ptr_register,
+              compiler->vars[k].size << compiler->loop_shift);
```

Comment:
For the readability, as they do with Linux kernel, if it's over 1 line (regardless of the number of statements), it'd be better to be enclosed by ```{``` and ```}```.

Reply:
Thx, updated.